django_project/lesson/templates/worksheet/print.html

```diff
@@ -213,6 +213,11 @@ <h2>{% trans 'Check your knowledge:' %}</h2>
         </div>
         <h2>{% trans 'Further reading:'%}</h2>
         <ul>
+            {% if worksheet.youtube_link %}
+                <li>
+                    <a target="_blank" href="{{ worksheet.youtube_link }}">module video tutorial</a>
```

Comment:
Can we change the casing on this to "Module video tutorial" and also expose it as a description, with the full link represented the same way other elements within the "Further reading" section are

Reply:
@zacharlie, I updated the PR, it looks like:

![1224_youtubelink](https://user-images.githubusercontent.com/40058076/102884707-0f239600-448d-11eb-8201-3266815575d7.png)